src/SharedWeb/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -176,6 +177,11 @@ public static void AddDefaultServices(this IServiceCollection services, GlobalSe
         services.AddSingleton<IStripeSyncService, StripeSyncService>();
         services.AddSingleton<IMailService, HandlebarsMailService>();
         services.AddSingleton<ILicensingService, LicensingService>();
+        services.AddSingleton<ILookupClient>((serviceProvider) =>
```

Comment:
Sorry, last comment! Since we aren't using the result of the expression, we can replace `serviceProvider` with an underscore `_`.

Ref: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards

Reply:
Good catch